osquery/dispatcher/scheduler.cpp

```diff
@@ -228,6 +234,22 @@ void SchedulerRunner::maybeFlushLogs(uint64_t time_step) {
   }
 }
 
+bool isDenylisted(const ScheduledQuery& query) {
```

Comment:
I feel confused by this function.

Or possible that there's an inversion in `!FLAGS_schedule_allow_denylisted_events`

Reply:
Any suggestions?